src/store/reducers/movieReducer.js

```diff
@@ -0,0 +1,51 @@
+import {
+  ADD_LIKES,
+  ADD_RATINGS,
+  SET_MOVIES_LIST,
+  SET_CURRENT_MOVIE,
+} from '../constants/constants';
+
+const initialState = {
+  moviesList: [],
+  currentMovie: null,
+};
+
+// eslint-disable-next-line default-param-last
```

Comment:
please, remove all eslint comments. Just remove rule if needed or update the code